django/contrib/admin/static/admin/js/admin/RelatedObjectLookups.js

```diff
@@ -118,7 +118,9 @@
     }
 
     function dismissAddRelatedObjectPopup(win, newId, newRepr) {
-        const name = removePopupIndex(win.name);
+        const urlParams = new URLSearchParams(win.location.search);
+        const tackOn = urlParams.has('_name_alteration') ? urlParams.get('_name_alteration') : '';
```

Comment:
This can be simplified to:
```suggestion
        const tackOn = urlParams.get('_name_alteration') || '';
```